berate_ap

```diff
@@ -705,6 +706,7 @@ ISOLATE_CLIENTS=0
 SHARE_METHOD=nat
 IEEE80211N=0
 IEEE80211AC=0
+IEEE80211W=0
```

Comment:
From what Ive seen is this is where defaults are set. and then the variable is just used later. probably leave this blank if 0 isnt the default.